operatorconfig/moduleconfig/replication/v1.10.0/controller.yaml

```diff
@@ -271,7 +271,7 @@ spec:
               value: /app/certs
             - name: X_CSI_REPLICATION_CONFIG_FILE_NAME
               value: config
-          image: quay.io/dell/container-storage-modules/dell-replication-controller:nightly # TODO: Change this image to latest released version once it is released
+          image: quay.io/dell/container-storage-modules/dell-replication-controller:v1.10.0
           imagePullPolicy: Always
```

Comment:
It would be better if the _imagePullPolicy_ is set to 'IfNotPresent' for versioned images.
This applies to other components as well.
CC @donatwork

Reply:
I agree, but it is too late to make more changes. We can tackle that first thing in CSM 1.13